views/subscription.py

```diff
@@ -46,20 +46,22 @@ def post(self, request, *args, **kwargs):
                 },
                 status=status.HTTP_400_BAD_REQUEST
             )
-
-        for category in new_subscriptions:
+        subscribe,unsubscribe = get_subscription(new_subscriptions,new_unsubscription,request.person,'notifications').get_should_subscribe()
```

Comment:
Just a suggestion, the get_subscription class can be made as a serializer, if you want

Reply:
Also, will it be better if placed in categories service?